CMakeLists.txt

```
@@ -108,3 +108,7 @@ endif()
 if(NOT (DJINNI_WITH_OBJC OR DJINNI_WITH_JNI))
   message(FATAL_ERROR "At least one of DJINNI_WITH_OBJC or DJINNI_WITH_JNI must be enabled.")
 endif()
+
+
+add_subdirectory(test-suite)
```

I think it would be good to make testing optional, 
E.g. for package building, it is not required and adds just some seconds more of cmake time
maybe like this, it allows passine a `-DBUILD_TESTING=OFF` to cmake and then the subfolder will not be added

```suggestion
include(CTest)
if (BUILD_TESTING)
  add_subdirectory(test-suite)
endif()
```